moveit_core/CMakeModules/FindBullet.cmake

```diff
@@ -0,0 +1,10 @@
+include(FindPackageHandleStandardArgs)
+find_package(PkgConfig)
+
+if(PKGCONFIG_FOUND)
+  pkg_check_modules(BULLET bullet)
```

Comment:
Shouldn't that be
```
pkg_check_modules(BULLET Bullet${Bullet_FIND_VERSION}>=2.77)
```

Reply:
the version is handled in the last line through `VERSION_VAR BULLET_VERSION` which makes it possible to use `find_package(Bullet 2.87)` in the `moveit_core/CMakeLists.txt`. I found that clearer to have the requirement for the Bullet version in the `moveit_core/CmakeLists.txt` instead of hidden inside the `FindBullet.cmake`.

Reply:
I missed that. That's even better.